tests/src/main/java/com/hazelcast/stabilizer/tests/map/MapEntryListenerTest.java

```diff
@@ -59,176 +73,200 @@
-                            eventCount.localUpdateCount.getAndIncrement();
-                        }
+                            break;
+                        case REPLACE:
+                            String oldValue = map.get(key);
+                            if (oldValue != null && map.replace(key, oldValue, value)) {
+                                eventCount.updateCount.getAndIncrement();
+                            }
+                            break;
+                        default:
+                            throw new UnsupportedOperationException();
                     }
-                } else if (chance < evictProb + writeProb) {
+                    break;
+                case EVICT:
                     map.lock(key);
                     try {
                         if (map.containsKey(key)) {
-                            eventCount.localEvictCount.getAndIncrement();
+                            eventCount.evictCount.getAndIncrement();
                         }
                         map.evict(key);
                     } finally {
                         map.unlock(key);
                     }
-                } else if (chance < removeProb + evictProb + writeProb) {
-                    Object o = map.remove(key);
-                    if (o != null) {
-                        eventCount.localRemoveCount.getAndIncrement();
+                    break;
+                case REMOVE:
+                    String oldValue = map.remove(key);
+                    if (oldValue != null) {
+                        eventCount.removeCount.getAndIncrement();
                     }
-                } else if (chance < deleteProb + removeProb + evictProb + writeProb) {
+                    break;
+                case DELETE:
                     map.lock(key);
                     try {
                         if (map.containsKey(key)) {
-                            eventCount.localRemoveCount.getAndIncrement();
+                            eventCount.removeCount.getAndIncrement();
                         }
                         map.delete(key);
                     } finally {
                         map.unlock(key);
                     }
-                }
+                    break;
+                default:
+                    throw new UnsupportedOperationException();
             }
-            IList results = targetInstance.getList(basename + "eventCount");
-            results.add(eventCount);
         }
-    }
 
-    @Verify(global = true)
-    public void globalVerify() throws Exception {
-        IList<EntryListenerImpl> listeners = targetInstance.getList(basename + "listeners");
-        for (int i = 0; i < listeners.size() - 1; i++) {
-            EntryListenerImpl a = listeners.get(i);
-            EntryListenerImpl b = listeners.get(i + 1);
-            assertEquals(basename + ": not same amount of event in all listeners", a, b);
```

Comment:
The method has just been moved between warmup and run, where I found it in most recent tests.